core/tabs/system-setup/remove-snaps.sh

```diff
@@ -3,27 +3,29 @@
 . ../common-script.sh
 
 removeSnaps() {
-    case "$PACKAGER" in
-        pacman)
-            "$ESCALATION_TOOL" "$PACKAGER" -Rns snapd
-            ;;
-        apt-get|nala)
-            "$ESCALATION_TOOL" "$PACKAGER" autoremove --purge snapd
-            if [ "$ID" = ubuntu ]; then
-                "$ESCALATION_TOOL" apt-mark hold snapd
-            fi
-            ;;
-        dnf)
-            "$ESCALATION_TOOL" "$PACKAGER" remove snapd
-            ;;
-        zypper)
-            "$ESCALATION_TOOL" "$PACKAGER" remove snapd
-            ;;
-        *)
-            printf "%b\n" "${RED}Unsupported package manager: ""$PACKAGER""${RC}"
-            exit 1
-            ;;
-    esac
+    if command_exists snap; then
+        case "$PACKAGER" in
+            pacman)
+                "$ESCALATION_TOOL" "$PACKAGER" -Rns snapd
+                ;;
+            apt-get|nala)
+                "$ESCALATION_TOOL" "$PACKAGER" autoremove --purge snapd
+                if [ "$ID" = ubuntu ]; then
+                    "$ESCALATION_TOOL" apt-mark hold snapd
+                fi
+                ;;
```

Comment:
```suggestion
    if command_exists snap; then
        case "$PACKAGER" in
            apt-get|nala)
                "$ESCALATION_TOOL" "$PACKAGER" autoremove --purge snapd
                if [ "$ID" = ubuntu ]; then
                    "$ESCALATION_TOOL" apt-mark hold snapd
                fi
                ;;
```

Snapd is not a valid package on arch linux
![image](https://github.com/user-attachments/assets/1487116b-8091-451c-9daf-eed90af45d17)

Reply:
It's a AUR package. AUR packages can be uninstalled using pacman

Reply:
ah i just noticed that it was in the aur